app/views/items/index.html.erb

```diff
@@ -1,3 +1,22 @@
+<h1>My Items</h1>
+
+<br>
+<section id="all_items-<%=@merchant.id%>">
 <% @items.each do |item| %>
-  <p><%= link_to item.name, merchant_item_path(@merchant.id, item.id) %></p> 
-<% end %>
\ No newline at end of file
+  <p><%= link_to item.name, merchant_item_path(@merchant.id, item.id) %></p>
+<% end %>
+</section>
+
+<section id="top_five_items-<%=@merchant.id%>">
+<h3>Top 5 Items:</h3>
+<ol>
+<% @merchant.top_five_items.each do |item| %>
+<div id="item<%=item.id%>">
+  <li><%= link_to item.name, merchant_item_path(@merchant.id, item.id) %> - <%= item.total_revenue %> in sales 
```

Comment:
hmmmmm, jsut not sure where this total_revenue method is stored. I also use a total revenue function in the invoices.rb. Maybe we could combine them and dry up our code a bit.

Reply:
@jbreit88 this total_revenue call is coming from the top_five_items method that uses an 'as' to set the function it is doing inside the method as 'total_revenue' so it can then be called in this view too. Not sure about extracting both cases out since this is tied to top_five_items, but if total_revenue is being found the same way then maybe we can!